server/channels/store/sqlstore/draft_store.go

```diff
@@ -236,3 +233,72 @@ func (s *SqlDraftStore) determineMaxDraftSize() int {
 
 	return maxDraftSize
 }
+
+func (s *SqlDraftStore) GetLastCreateAtAndUserIdValuesForEmptyDraftsMigration(createAt int64, userId string) (int64, string, error) {
+	var drafts []struct {
+		CreateAt int64
+		UserId   string
+	}
+
+	query := s.getQueryBuilder().
+		Select("CreateAt", "UserId").
+		From("Drafts").
+		Where(sq.Or{
+			sq.Gt{"CreateAt": createAt},
+			sq.And{
+				sq.Eq{"CreateAt": createAt},
+				sq.Gt{"UserId": userId},
+			},
+		}).
+		OrderBy("CreateAt", "UserId ASC").
```

Comment:
Does it matter to have `ASC` for either actually? Been a while since I've played in SQL but it should sort strings alphabetically and integers in numerical order by default right?

Reply:
Yeah, I'd be fine with removing it altogether :)